src/audio/dai.c

```diff
@@ -55,7 +55,7 @@
 
```

Comment:
@akloniex Please explain in the commit message why this commit is needed. I understand that this is very clear for you now, but people reading the commit message history one year from now will be very confused.